keyboards/kelwin/keymaps/via/keymap.c

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Kelwin
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
Just a heads up, you may want to have a `LAYOUT_all`, so that all of the keys for alternate layouts can be populated by default, for the via config.